### DISCUSSION TITLE
Redefine loop variables in different cells

### DISCUSSION THREAD

**Discussion by TAOtxi** [marimo-team/marimo#2127] [Ideas]
The example is in https://static.marimo.app/static/problem-j30k

I think loop variable can be redefined in different cells..

**Comment by akshayka** [comment]
Prefix a variable with an underscore to make it local to a cell: https://marimo.app/l/8fwxil

Learn more at docs.marimo.io (search for local variables).

For complications on relaxing uniqueness constraints, you can learn more by reading #1477
